apps/marketing/src/components/(marketing)/hero.tsx

```diff
@@ -111,11 +111,11 @@ export const Hero = ({ className, ...props }: HeroProps) => {
           <Button
             type="button"
             variant="outline"
-            className="rounded-full bg-transparent backdrop-blur-sm"
+            className="rounded-full bg-transparent backdrop-blur-sm max-[320px]:h-auto max-[320px]:flex-col max-[320px]:gap-1"
             onClick={onSignUpClick}
           >
             Get the Early Adopters Plan
-            <span className="bg-primary dark:text-background -mr-2.5 ml-2.5 rounded-full px-2 py-1.5 text-xs">
+            <span className="bg-primary dark:text-background rounded-full px-2 py-1.5 text-xs max-[320px]:w-full min-[375px]:-mr-2.5 min-[375px]:ml-2.5">
               $30/mo. forever!
             </span>
           </Button>
```

Comment:
You should be able to fit these components on one line like this, instead of having the `$30/mon...` overflow under the button.

![image](https://github.com/documenso/documenso/assets/20962767/6e77b9d8-a7a7-494b-80b9-761e67bf1e4d)

The above example was 320px

You can use a combination of `text-xs`, smaller margins/padding and negative margins on the button to provide more space.

Could you also fiddle with the other ones to they stay on one line?

Reply:
yup